apis/core/v1alpha1/flagsourceconfiguration_types.go

```diff
@@ -92,6 +94,11 @@ type FlagSourceConfigurationSpec struct {
 	// DefaultSyncProvider defines the default sync provider
 	// +optional
 	DefaultSyncProvider SyncProviderType `json:"defaultSyncProvider"`
+
+	// DefaultLogFormat allows for the sidecar log format to be overridden, defaults to 'json'
+	// +optional
+	// +kubebuilder:default:=json
+	DefaultLogFormat string `json:"logFormat"`
```

Comment:
this change also should be introduced into the `v1alpha2` version, with the appropriate changes made in the `flagsourceconfiguration_conversion.go` file.

Reply:
done